python/dockerfile

```
@@ -1,4 +1,15 @@
-FROM alpine
+FROM python:3.8-slim-buster
 
-# STEP4-4では以下は変更しない
-CMD ["python", "-V"]
+COPY python/main.py python/main.py
+
+COPY python/image python/image
+
+copy db ./db
```

```suggestion
COPY db ./db
```

細かいですが、ここだけ小文字になっています！